extensions/wikia/AdEngine/js/AdLogicPageViewCounter.js

```diff
@@ -49,6 +59,7 @@ define('ext.wikia.adEngine.adLogicPageViewCounter', [
 	}
```

Comment:
Now that we have a separate get and increment methods let's not return the value from the increment method.